sdk/core/azure-core/src/main/java/com/azure/core/models/spatial/BoundingBox.java

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.models.spatial;
+
+import com.azure.core.annotation.Immutable;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * A list of points representing the quadrilateral bounding box. The points are listed in clockwise
+ * order: top-left, top-right, bottom-right, bottom-left.
+ */
+@Immutable
+public final class BoundingBox {
```

Comment:
Thoughts on renaming this to bounding area? Current implementation doesn't require this to be a box.

Reply:
@kristapratico @annelo-msft Thoughts on renaming this to `BoundingArea` ?
 I know with FR's current implementation it will always have 4 points but we wanted to keep this extendable to future-proof service team supporting more than 4 points.

Reply:
I'm not too opinionated on this - but confused why `box` isn't correct. If it has 4 corners that seems like a box to me?

I'm not sure how other vision services use BoundingBox or how their points are represented - Anne might know more. If more than 4 points can be returned then definitely BoundingArea or BoundingRegion could work IMO.

Reply:
Based on the current implementation there could be more, or less, than 4 points. If we want to make this into a quadrilateral I'd rather see us take 4 points in the constructor.